modules/concepts/hooks/list-of-hooks/actionAddressGridDefinitionModifier.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: actionAddressGridDefinitionModifier
+hidden: true
+hookTitle: 'Modify address grid definition'
+files:
+    -
+        url: 'https://github.com/PrestaShop/PrestaShop/blob/9.0.x/'
+        file: 
+locations:
+    - 'back office'
+type: action
+hookAliases: 
+array_return: false
+check_exceptions: false
+chain: false
+origin: core
+description: 'This hook allows to alter address grid columns, actions and filters'
+
+---
+
+{{% hookDescriptor %}}
+
+## Call of the Hook in the origin file
+
+```php
+;
+```
```

Comment:
This looks like a mistake. Again, it would be good to have a source for this hook here.